src/backup/backup_creator.py

```diff
@@ -20,9 +20,10 @@ def create_backup(self, source_table_entity, bq_table_metadata):
         logging.info('Scheduling copy job for backup, request correlation id:'
                      ' %s', request_correlation_id.get())
 
+        table_location = bq_table_metadata.get_location()
```

Comment:
We had an issue some time ago, that small tables (under 10kB or sth like that) doesn't have location specified as they are stored together with metadata. We should check if that's still the case.

Reply:
in this particular case: bq_table_metadata.get_location() was used earlier, but was inline into method invocation, I only extract it to the variable and reuse it in another method. 

So question is, do we suffer from it in last few months?

Reply:
Currently it's ok, as it's either US or EU. We need to test this with other regions